src/java/picard/fingerprint/HaplotypeProbabilityOfNormalGivenTumor.java

```diff
@@ -1,7 +1,7 @@
 /*
  * The MIT License
  *
- * Copyright (c) 2010 The Broad Institute
+ * Copyright (c) 2016 The Broad Institute
```

Comment:
hmmm. The original version was not written in 2010...that's just a typo. I think it's fine as 2016...